src/Renderer/PointsMaterial.js

```diff
@@ -9,7 +9,13 @@ export const PNTS_MODE = {
     COLOR: 0,
     INTENSITY: 1,
     CLASSIFICATION: 2,
-    NORMAL: 3,
+    ELEVATION: 3,
+    RETURN_NUMBER: 4,
+    RETURN_TYPE: 5,
+    RETURN_COUNT: 6,
+    PTID: 7,
```

Comment:
I would prefer `POINT_ID` or `POINT_SOURCE_ID`:
```suggestion
    POINT_ID: 7,
```